ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -851,20 +997,27 @@ export const RuntimePanel = fp.flow(
     computeType: initialCompute,
     machine: initialMasterMachine,
     diskSize: diskSize,
```

Comment:
I'm thinking that the cleaner model here may be to have the singular disk size, then have a separate variable about whether or not the PD is reattachable (or inverse: autodeleting).

Reply:
For now, the diskSize variable is also reused in other pannels, like Dataproc. The user might edit it in Dataproc and the modification will still be showed when he changes back to Standard mode. Though we can add some conditional logic to avoid that, I'm thinking it might be difficult to maintain that if we'll add more pannel types in the future.

Reply:
I think this behavior is desirable. The components are the same on the page. If I change my compute type, the desired behavior is for the main machine type and main disk to remain unchanged

Reply:
Per slack discussion - separate variables are OK because the PD goes elsewhere on the page, though I don't think it's terrible still if they share state.